### ISSUE TITLE
Greenland whole column instability at T255

### ISSUE THREAD

**Issue by milankl** [SpeedyWeather/SpeedyWeather.jl#289]
With 
```julia
run_speedy(Float32,PrimitiveDryCore,verbose=false,initial_conditions=StartFromFile(),restart_id=23,trunc=255,n_days=20,output=true,output_dt=1,Grid=OctahedralClenshawGrid,nlev=31)
```
There's an instability after about 12 days that spans the whole column, level 15

![image](https://user-images.githubusercontent.com/25530332/230497918-51a5ad93-1398-4161-8580-520ae522b605.png)

It's barely visible in surface or stratospheric temperature, nor surface pressure. But also mid-tropospheric temperature

![image](https://user-images.githubusercontent.com/25530332/230498640-4cdf4634-8fb3-464f-91bd-8eccffb94e04.png)

and (particularly?) pronounced in vorticity a bit higher (level 10)

![image](https://user-images.githubusercontent.com/25530332/230499285-cea5d84c-8fd5-47db-ad07-69eafb6e55d4.png)

**Comment by milankl** [comment]
Hypothesis: The instability is created because the extra Laplacian in the stratosphere reaches in this model setup to level k=9 but not below. So there's an abrupt change in the horizontal diffusion between k=9,10. Cutting through the instability at about lon=292˚E, sigma coordinates on y but upside down.

![image](https://user-images.githubusercontent.com/25530332/230531671-e63be0e1-b9bf-4d22-93c2-d687dbc9e9bf.png)

**Comment by milankl** [comment]
Another one of those, this time at the coast of Antarctica

![image](https://user-images.githubusercontent.com/25530332/230971026-27b49c9a-aad4-4212-91a3-ed29acddecb4.png)

run with

```julia
run_speedy(Float32,PrimitiveDryCore,verbose=false,initial_conditions=StartFromFile(),restart_id=23,trunc=255,n_days=30,output=true,output_dt=1,output_vars=[:vor],Grid=OctahedralGaussianGrid,nlev=31)
```

after #290 got merged

**Comment by milankl** [comment]
Either disabling the Laplacian with tapering from sigma 0,0.2 or setting its power to 2 (biharmonic) does seem to resolve the issue (level 20)

![image](https://user-images.githubusercontent.com/25530332/231001167-9872d061-968e-47d0-834f-baab9c5cc3e1.png)

run with

```julia
run_speedy(Float32,PrimitiveDryCore,verbose=false,initial_conditions=StartFromFile(),restart_id=31,trunc=255,n_days=10,output=true,output_dt=1,output_vars=[:vor,:temp],Grid=OctahedralGaussianGrid,nlev=31,diffusion=HyperDiffusion(tapering_σ=0))
```

suggestion therefore to simply change the default diffusion settings

**Comment by milankl** [comment]
New diffusion scheme is merged with #298 but maybe the problem here is related to #299 hence could be resolved when #302 is merged (and set as new default)
